src/lib.rs

```diff
@@ -185,34 +337,58 @@ mod tests {
         );
     }
 
-    /// Test case involving a missing module
+    /// Test case involving missing and invalid modules
     #[test]
     fn missing_module() {
         let mut env = TestResolver::default();
-        env.register("src/lib.rs", "mod missing;");
+        env.register("src/lib.rs", "mod missing;\nmod invalid;");
+        env.register("src/invalid.rs", "this-is-not-valid-rust!");
 
-        let result = InlinerBuilder::default()
-            .error_not_found(true)
-            .parse_internal(Path::new("src/lib.rs"), env);
+        let result = InlinerBuilder::default().parse_internal(Path::new("src/lib.rs"), env);
 
         match result {
-            Err(Error::NotFound(errors)) => {
+            Ok(r) if r.has_errors() => {
```

Comment:
This line appears to fail on rustc 1.32. Changing the example not to offend should be easy enough, but I thought NLL were available starting in 1.31.0 so I'm not sure why this is failing.